packages/aws/changelog.yml

```
@@ -1,4 +1,18 @@
 # newer versions go on top
+- version: "3.12.0"
+  changes:
+    - description: Mapping changes in `inspector` datastream for Cloud Detection and Response (CDR) vulnerability workflow.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/14306
+    - description: Parse and map newly introduced fields in the `inspector` data stream.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/14306
+    - description: Enable request trace log removal.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/14306
+    - description: Remove metadata fields added by the Agentless policy.
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/14306
```

```suggestion
    - description: Enable Agentless deployment.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/14306
```

Its not going to be a bugfix since the Agentless is not enabled yet.